context.go

```diff
@@ -67,6 +65,65 @@ type SpanContext struct {
 	//
 	// See JaegerDebugHeader in constants.go
 	debugID string
+
+	// samplingState is shared across all spans
+	samplingState *samplingState
+}
+
+type samplingState struct {
+	_flags atomic.Int32 // Only lower 8 bits are used. We use an int32 instead of a byte to use CAS operations
```

Comment:
we never use underscores for field names, what's the reason to start now?

Reply:
I used it to avoid a conflict with `flags()`, but we can rename to `ctxFlags` or sth else

Reply:
oic. Usually collisions are avoided by making methods public, like Flags.

Let's keep the underscore.